src/crab/ebpf_transformer.cpp

```diff
@@ -982,13 +982,17 @@ static void overflow_bounds(NumAbsDomain& inv, variable_t lhs, number_t span, in
 static void overflow_signed(NumAbsDomain& inv, const variable_t lhs, const int finite_width) {
     const auto span{finite_width == 64   ? number_t{std::numeric_limits<uint64_t>::max()}
                     : finite_width == 32 ? number_t{std::numeric_limits<uint32_t>::max()}
+                    : finite_width == 16 ? number_t{std::numeric_limits<uint16_t>::max()}
+                    : finite_width == 8  ? number_t{std::numeric_limits<uint8_t>::max()}
                                          : throw std::exception()};
     overflow_bounds(inv, lhs, span, finite_width, true);
 }
 
 static void overflow_unsigned(NumAbsDomain& inv, const variable_t lhs, const int finite_width) {
     const auto span{finite_width == 64   ? number_t{std::numeric_limits<uint64_t>::max()}
                     : finite_width == 32 ? number_t{std::numeric_limits<uint32_t>::max()}
+                    : finite_width == 16 ? number_t{std::numeric_limits<uint16_t>::max()}
+                    : finite_width == 8  ? number_t{std::numeric_limits<uint8_t>::max()}
                                          : throw std::exception()};
     overflow_bounds(inv, lhs, span, finite_width, false);
 }
```

Comment:
To avoid repetition, change `overflow_bound()` to accept width:
```c++
static void overflow_bounds(NumAbsDomain& inv, variable_t lhs, int finite_width, bool issigned) {
    using namespace crab::dsl_syntax;
    auto interval = inv[lhs];
    if (interval.size() >= interval_t::unsigned_int(finite_width).size()) {
...
```

And simplify here:
```suggestion
static void overflow_signed(NumAbsDomain& inv, const variable_t lhs, const int finite_width) {
    overflow_bounds(inv, lhs, finite_width, true);
}

static void overflow_unsigned(NumAbsDomain& inv, const variable_t lhs, const int finite_width) {
    overflow_bounds(inv, lhs, finite_width, false);
}
```

Reply:
Applied.